ballerina/tests/file-test.bal

```diff
@@ -77,6 +78,19 @@ function testCopyDir() {
     }
 }
 
+@test:Config {}
+function testCopyDir2() returns error? {
+    string targetPath = "tests/resources/temp-dir/nested-file.txt";
+    error? copyResult = copy(srcDir, "tests/resources/temp-dir", REPLACE_EXISTING);
+    if copyResult is error {
+        test:assertFail("Directory not copied!");
+    } else {
+        string readContent = check io:fileReadString(targetPath);
+        test:assertEquals(readContent, "Hi");
+        check io:fileWriteString(targetPath, "");
```

Comment:
shall we just remove the file, instead making the content empty

Reply:
do we need an empty file in the target directory always

Reply:
Shall we have at least two files in the src dir

Reply:
This comment is not valid. The issue only happens when we try to replace existing files

Reply:
I have added [this test by creating files](https://github.com/ballerina-platform/module-ballerina-file/pull/429/commits/e471c70b7b72d1025056dcfd9e98910c789c8713#diff-66a9555be220b2129eec3c611997479d1e31ecd1f9a93ad74fe37b5a3a3c6cabR83). It was failed in the GitHub workflow, but the local build succeeded.